app/src/main/java/org/fossasia/phimpme/leafpic/util/AlertDialogsHelper.java

```diff
@@ -61,6 +64,35 @@ public static AlertDialog getInsertTextDialog(final ThemedActivity activity, Ale
         dialogBuilder.setView(dialogLayout);
         return dialogBuilder.create();
     }
+    public static AlertDialog getInsertTextDialogLink(final ThemedActivity activity, AlertDialog.Builder dialogBuilder , EditText editText, String link) {
```

Comment:
@Subhankar29, Why don't you use this function from AlertDialogHelper.java class(https://github.com/fossasia/phimpme-android/blob/development/app/src/main/java/org/fossasia/phimpme/leafpic/util/AlertDialogsHelper.java#L37).
It is a static function and you can call it from anywhere. Why are you creating the same function again? Please avoid creating duplicate functions.